src/components/Header/index.tsx

```diff
@@ -13,12 +13,27 @@ import React, { FC, useState } from 'react';
 import AccountCircle from '@mui/icons-material/AccountCircle';
 import CheckIcon from '@mui/icons-material/Check';
 import EmailIcon from '@mui/icons-material/Email';
+import TextField from '@mui/material/TextField';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogTitle from '@mui/material/DialogTitle';
 import Switch from '@mui/material/Switch';
 import { NavLink } from 'react-router-dom';
 import { BoxStyled, SecondaryTextWithBorderStyled, LoginTypographyStyled, LogoutStyled } from './styles';
 import { IUserInfo } from './interfaces';
 
 export const Header: FC = () => {
+  const [isOpen, setIsOpen] = useState(false);
```

Comment:
everything is ok. but it can be even better following single responsibility principle. move feeback handlers, icon and dialog to components. and import it here